qdrant-landing/content/documentation/guides/distributed_deployment.md

```diff
@@ -106,6 +122,18 @@ Example result:
 }
 ```
 
+## Enabling distributed mode in Qdrant Cloud
+
+In Qdrant Cloud, when you click "Scale Up" to increase your cluster size to >1, the distributed mode settings are set automatically, resulting in a new empty node starting up. For best results, ensure your cluster is running Qdrant v1.7.4 or higher.
```

Comment:
I've made some assumptions here:

```suggestion
In Qdrant Cloud, when you "Scale Up" to increase set up more than one cluster, the distributed mode settings are set automatically. A new empty node starts up. For best results, ensure your cluster is running Qdrant v1.7.4 or higher.
```

Reply:
Will make this clearer by explicitly pointing the user to the next section 👍